cassandra-launcher/src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala

```diff
@@ -82,6 +82,14 @@ object CassandraLauncher {
     port
   }
 
+  @tailrec
+  private def freePort(usedPort: Int): Int = {
+    val port = freePort()
```

Comment:
possibly limit the tries? Tho i can't see why it would keep picking the same port